src/main/resources/tanagra/cms_synpuf/expanded/cms_synpuf.json

```diff
@@ -3,19 +3,19 @@
\"key\": \"id\", \"width\": 120, \"title\": \"Concept ID\" },         { \"key\": \"t_rollup_count\", \"width\": 120, \"title\": \"Roll-up count\" }       ],       \"occurrence\": \"ingredient_occurrence\",       \"classification\": \"ingredient\"     },     {       \"type\": \"attribute\",       \"id\": \"tanagra-ethnicity\",       \"title\": \"Ethnicity\",       \"category\": \"Program data\",       \"attribute\": \"ethnicity\"     },     {       \"type\": \"attribute\",       \"id\": \"tanagra-gender\",       \"title\": \"Gender identity\",       \"category\": \"Program data\",       \"attribute\": \"gender\"     },     {       \"type\": \"attribute\",       \"id\": \"tanagra-race\",       \"title\": \"Race\",       \"category\": \"Program data\",       \"attribute\": \"race\"     },     {       \"type\": \"attribute\",       \"id\": \"tanagra-sex_at_birth\",       \"title\": \"Sex assigned at birth\",       \"category\": \"Program data\",       \"attribute\": \"sex_at_birth\"     },     {       \"type\": \"attribute\",       \"id\": \"tanagra-year_of_birth\",       \"title\": \"Year of birth\",       \"category\": \"Program data\",       \"attribute\": \"year_of_birth\"     }   ],   \"demographicChartConfigs\": {     \"groupByAttributes\": [\"gender\", \"race\", \"year_of_birth\"],     \"chartConfigs\": [       {         \"title\": \"Gender identity\",         \"primaryProperties\": [{ \"key\": \"gender\" }]       },       {         \"title\": \"Gender identity, Current age, Race\",         \"primaryProperties\": [           { \"key\": \"gender\" },           {             \"key\": \"age\",             \"buckets\": [               {                 \"min\": 18,                 \"max\": 45,                 \"displayName\": \"18-44\"               },               {                 \"min\": 45,                 \"max\": 65,                 \"displayName\": \"45-64\"               },               {                 \"min\": 65,                 \"displayName\": \"65+\"               }             ]           }         ],         \"stackedProperty\": { \"key\": \"race\" }       }     ]   },   \"prepackagedConceptSets\": [     {       \"id\": \"_demographics\",       \"name\": \"Demographics\",       \"occurrence\": \"\"     },     {       \"id\": \"_analgesics\",       \"name\": \"Analgesics\",       \"occurrence\": \"ingredient_occurrence\",       \"filter\": {         \"type\": \"CLASSIFICATION\",         \"occurrenceID\": \"ingredient_occurrence\",         \"classificationID\": \"ingredient\",         \"keys\": [21604253]       }     }   ],   \"criteriaSearchConfig\": {     \"criteriaTypeWidth\": 120,     \"columns\": [       { \"key\": \"name\", \"width\": \"100%\", \"title\": \"Concept Name\" },       { \"key\": \"vocabulary_t_value\", \"width\": 120, \"title\": \"Vocab\" },       { \"key\": \"concept_code\", \"width\": 120, \"title\": \"Code\" },       { \"key\": \"t_rollup_count\", \"width\": 120, \"title\": \"Roll-up Count\" }     ]   } } ",
```

Comment:
Anything I should look at particularly in this changelog?

Reply:
No, and I would ignore everything in `expanded` since it's automatically generated by `EXPAND_CONFIG`. Only look at changes in `cms_synpuf/original/`.